templates/main/00_struct.go.tpl

```diff
@@ -71,6 +74,8 @@ var {{$alias.UpSingular}}TableColumns = struct {
 	{{end -}}
 }
 
+const {{$alias.UpSingular}}TableName = "{{.Table.Name}}"
```

Comment:
This is unnecessary, there is a struct with table names in the singleton. Let's reuse that.

Reply:
this is fixed